package.json

```diff
@@ -1,15 +1,25 @@
 {
   "name": "salesforce-jwt-promise",
-  "version": "0.1.8",
+  "version": "1.2.0",
   "description": "Promise Based SalesForce Auth 2.0 JWT Bearer Token Flow Implementation, written in typescript",
   "main": "./bin/index.js",
   "typings": "bin/index.d.ts",
   "scripts": {
-      "prepublish": "tsc"
+    "prepublish": "./node_modules/.bin/tsc"
```

Comment:
Is this compatible with MS Windows machines? (I can't check it but maybe anyone can do it)